manifests/m/Microsoft/VisualStudio/Locator/3.0.3.9285/Microsoft.VisualStudio.Locator.installer.yaml

```
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.Locator
+PackageVersion: 3.0.3.9285
+Installers:
+- Architecture: x86
+  InstallerType: portable
+  InstallerUrl: https://github.com/microsoft/vswhere/releases/download/3.0.3/vswhere.exe
+  InstallerSha256: 261D281CE9C95C3E46900E67ACE8C3518DD7B2596774F63C92E2BB1FBCE71D4C
+  Platform:
+    - "Windows.Desktop"
+  InstallModes:
+    - "silent"
+  ReleaseDate: "2022-06-02"
+ManifestType: installer
+ManifestVersion: 1.1.0
```

```suggestion
ManifestVersion: 1.2.0
```
`1.2.0` schema includes support for `InstallerType: portable`